plugins/integrations/kubernetes-service/src/main/resources/conf/k8s-control-node.yml

```diff
@@ -320,6 +320,7 @@ write_files:
 
 runcmd:
   - chown -R cloud:cloud /home/cloud/.ssh
+  - ip link set dev eth0 mtu 1400
```

Comment:
Should we keep this as a general step?

Reply:
I am not sure what to do with this.. ? may be remove it, but try cks testing on physical env as opposed to nested?